ndn-svs/svspubsub.cpp

```diff
@@ -59,8 +59,12 @@ SVSPubSub::publish(const Name& name, const uint8_t* value, const size_t length,
       auto segmentName = Name(name).appendVersion(0).appendSegment(i);
       auto segment = Data(segmentName);
       segment.setFreshnessPeriod(freshnessPeriod);
-      segment.setContent({value + i * MAX_DATA_SIZE,
```

Comment:
`ndn::make_span()` instead of `{...}` should've fixed it

Reply:
Got it. I'm unfamiliar with this syntax.

Reply:
Done

Reply:
thanks